crates/cdk-mint-rpc/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "cdk-mint-rpc"
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+homepage = "https://github.com/cashubtc/cdk"
+repository = "https://github.com/cashubtc/cdk.git"
+rust-version = "1.63.0"                            # MSRV
+
+[dependencies]
+anyhow = "1"
+cdk = { path = "../cdk", version = "0.6.0", default-features = false, features = [
+    "mint",
+] }
+clap = { version = "~4.0.32", features = ["derive"] }
+# tonic = { version = ">=0.9, <0.12", features = [
+#     "channel",
+#     "tls",
+#     "tls-webpki-roots",
+# ] }
+tonic = { version = "0.9", features = [
+    "channel",
+    "tls",
+    "tls-webpki-roots",
+] }
```

Comment:
We don't want to enforce such an old version of `tonic`, `prost` and `tonic-build` so we should be able to support the range to enable support of msrv and newer rust version.